admin_manual/configuration/server/occ_command.rst

```diff
@@ -780,7 +781,7 @@ In the example above, the user_id ``alice`` is determined implicitly from the pa
   Mounts are only scannable at the point of origin. Scanning of shares including federated shares 
   is not necessary on the receiver side and therefore not possible.
 
-The ``--path``, ``--all`` and ``[user_id]`` parameters are exclusive - only one must be specified.
+The ``--path``, ``--all``, ``--groups`` and ``[user_id]`` parameters are exclusive - only one must be specified.
```

Comment:
Note: nothing bad happens if you specify lots of these together, you just end up getting whichever one the code implementation happens to check for first.

Reply:
Good to know, pity that the result seems somewhat random.